### DISCUSSION TITLE
SSO Redirection

### DISCUSSION THREAD

**Discussion by Santoshactive147** [inventree/InvenTree#8659] [Q&A] [answered]
![Screenshot (22)](https://github.com/user-attachments/assets/1477360b-94b1-47a2-a7af-1f3d32be69b8)
![Screenshot (23)](https://github.com/user-attachments/assets/1a5d2738-5f16-45cd-82aa-9eaf8d04b8d0)

when i am trying to sign in , its redirecting verfication , but trying to create new user , i want to redirect index page

**Comment by SchrodingersGat** [comment]
Please provide full version information for your InvenTree installation

**Comment by Santoshactive147** [comment, marked as answer]
@SchrodingersGat 
Thanks for reply !
Its Done , i just modified one line , make sure its taking in string 

def _redirect_strict_samesite(self, request, provider):

in this function  i just changed this =>  samesite_value = str(settings.SESSION_COOKIE_SAMESITE).lower()
